### PR TITLE
Improve snippets test coverage for toolingApi, testKit, and scala

### PR DESCRIPTION
**Pull request by 6hundreds**
Part of https://github.com/gradle/gradle/issues/21027

### CODE REVIEW

**Comment by 6hundreds** [comment]
@bot-gradle test this

File: subprojects/docs/src/snippets/toolingApi/eclipse/tests/sanityCheck.sample.conf [hunk 1]
```diff
@@ -1,2 +1,2 @@
 executable: gradle
-args: tasks
+args: build
```
Comment (mlopatkin) [inline subprojects/docs/src/snippets/toolingApi/eclipse/tests/sanityCheck.sample.conf:2]: Why is this a sanity check test?

Comment (6hundreds) [reply subprojects/docs/src/snippets/toolingApi/eclipse/tests/sanityCheck.sample.conf:2]: Yeah, it should be reverted as we discussed

**Comment by 6hundreds** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
There seems to be a few test failures: https://builds.gradle.org/buildConfiguration/Gradle_Master_Check_Stage_ReadyforNightly_Trigger/58091029?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildTestsSection=true

**Comment by 6hundreds** [comment]
@bot-gradle test this

**Review by mlopatkin — APPROVED** [review]
lgtm!

**Comment by 6hundreds** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Review by bamboo — COMMENTED** [review]
👍

**Pull request merged**